server/routers/jobs_router.py

```
@@ -421,10 +448,83 @@ async def load_offers(
+                headers=headers,
+                timeout=30,
+            )
+            resp.raise_for_status()
+            data = resp.json()
+            # Getting offers info from france travail API
+            liste_offres = get_offers(codeROME)
+            salaire = []
+            for offre in liste_offres:
+                salaire.append(calcul_salaire(str(offre.get('salaire').get('libelle')), str(offre.get('dureeTravailLibelle'))))
+            
+        except:
```

Ruff (0.14.13)

Do not use bare `except`

(E722)